src/storage/txn/scheduler.rs

```diff
@@ -740,220 +742,274 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
             };
 
             task.cmd
-                .process_write(snapshot, context)
+                .process_write(snapshot.clone(), context)
                 .map_err(StorageError::from)
         };
-        match deadline
+        let WriteResult {
+            ctx,
+            mut to_be_write,
+            rows,
+            pr,
+            lock_info,
+            lock_guards,
+            response_policy,
+        } = match deadline
```

Comment:
Besides deadline check do we have other situations the `high_priority_pool` will be used? There're some ddl transation options which would set the command priority but they're rarely used.

Reply:
resolve-lock requests use `high_priority_pool`.